src/Magnum/Math/Geometry/Test/DistanceTest.cpp

```diff
@@ -156,6 +158,20 @@ void DistanceTest::lineSegmentPoint3D() {
                     Constants::sqrt2());
 }
 
+void DistanceTest::pointPlaneUnnormalized() {
+    Vector3 point{1.0f, 1.0f, 1.0f};
+    Vector4 plane{2.0f, 2.0f, 2.0f, 0.0f};
```

Comment:
I would use a different value here so the difference between normalized and not normalized implementation is properly tested.